### PR TITLE
Fixed findbugs issues

### PR DESCRIPTION
**Pull request by rsandell**
And enabled FindBugs xml output so Jenkins can pick it up

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -64,6 +64,11 @@
       <version>1.0.2</version>
       <optional>true</optional>
     </dependency>
+    <dependency>
+      <groupId>com.google.code.findbugs</groupId>
+      <artifactId>annotations</artifactId>
+      <version>3.0.0</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Should be 3.0.1 IMO - it contains more signatures

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by varmenise** [comment]
:bee:

**Comment by stephenc** [comment]
(as plugin maintainer) :+1:

**Pull request merged**